flexmeasures/ui/templates/base.html

```diff
@@ -358,10 +357,14 @@
         });
     }
 
+    var combineLegend = 'true';
     {% if active_page == "assets" %}
         var dataPath = '/api/v3_0/assets/' + {{ asset.id }};
         var dataDevPath = '/api/dev/asset/' + {{ asset.id }};
         var datasetName = 'asset_' + {{ asset.id }};
+        {% if asset.sensors_to_show | length > 1 %}
```

Comment:
So basically combineLegend id always false.
We could also use that as default, then.
We also discussed counting the sensors and combine until their number reaches 7. Is that maybe too much work ?

Reply:
Challenge accepted.